src/h3lib/lib/h3Index.c

```diff
@@ -61,7 +61,9 @@ static char *H3ErrorDescriptions[] = {
  * @return The string describing the H3Error
  */
 const char *H3_EXPORT(describeH3Error)(H3Error err) {
-    if (err >= 0 && err <= 15) {  // TODO: Better way to bounds check here?
+    // err is always non-negative because it is an unsigned integer
+    // TODO: Better way to bounds check here?
+    if (ALWAYS(err >= 0) && err <= 15) {
```

Comment:
Do we need the negative number bounds check at all, then? It's literally impossible to assign a negative number to an `H3Error`, so I would consider that bounds check a silly mistake and drop it entirely.

Reply:
That's fair -- we may as well just remove it.